scripts/docker/install_from_docker_context_files.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# shellcheck disable=SC2086
+
+# Installs airflow and provider packages from locally present docker context files
+# This is used in CI to install airflow and provider packages in the CI system of ours
+# The packages are prepared from current sources and placed in the 'docker-context-files folder
+# Then both airflow and provider packages are installed using those packages rather than
+# PyPI
```

Comment:
A check would be helpful here as well, because it's very easy to get confused when scripts communicate through environment variables.

Reply:
Yep. I thoutght about doing it differently, but this is the easiest and more straightforward way in docker file. We could pass them by parameters, but they are not named, so that it is even worse.